shell.nix

```
@@ -28,7 +28,7 @@ let
     inputs;
 
   chromeCommand = if stdenv.isDarwin then "/Applications/Google Chrome.app/Contents/MacOS/Google Chrome" else "google-chrome";
-  # define shell startup command with special handling for OSX
+  # define shell startup command with special handling for macOS
```

```suggestion
  # define shell startup command with special handling for darwin
```
Let's split the difference ;)